docs_user/modules/proc_migrating-ceph-post.adoc

```diff
@@ -0,0 +1,27 @@
+[id="updating-the-cluster-mgr-configuration_{context}"]
+
+= Updating the {CephCluster} cluster mgr dashboard configuration
```

Comment:
@klgill right now, as per our conversation, the post section is about fixing leftovers from the previous deploy. For now I'm only thinking about the dashboard failover, but TLS might be a good candidate to extend this section. Considering it's day 2 and I don't have a specific procedure, I would just keep this and extend it when we have more content.

Reply:
I think we should avoid using "mgr" in the title. Would "Updating the {CephCluster} cluster with Ceph Manager modules that include Ceph Dashboard"  make sense?  I know that's a mouthful...

Reply:
Also, I want to confirm the sequence of events. Would customers perform this procedure immediately after the "Verifying the Red Hat Ceph Storage cluster after Ceph Monitor migration" procedure?

Reply:
yes about the sequence, customers are supposed to perform this part right after checking the cluster status.
Regarding the title, I think that suggestion would change the meaning here.
Basically we're going to update the Ceph dashboard configuration, I think we don't have to specify that is Ceph Manager module.
I'll push an update and let me know if it's better than what we currently have.

Reply:
Regarding the sequence, would it be more logical to put this procedure as the last step in the "Migrating Ceph Monitor daemons to Red Hat Ceph Storage nodes" assembly?

Reply:
When I think about a "post procedure" I imagine something generic that has the Dashboard update right now, but it might be extended to something else. For this reason, given this is a standalone procedure, it's part of the main migration guide, while adding in the migrate monitor daemons assembly let me thing that is somehow related to that section, while it's not.
If this is a concern feel free to reorganize the way you think it's better, I just based the location choice on a logical view of the main procedure.

Reply:
OK thank you for the explanation. We can leave it where it is.